server/models/helpers/NotificationHelper.ts

```diff
@@ -17,6 +17,11 @@ import {
 import { can } from "@server/policies";
 import { ProsemirrorHelper } from "./ProsemirrorHelper";
 
+type Recipient = {
```

Comment:
Pretty much all the changes relate to changing this interface from a simple `User[]` return to one that also includes the subscription, if present